lib/bot.js

```diff
@@ -162,9 +162,14 @@ class Bot {
       const nickname = Bot.getDiscordNicknameOnServer(author, fromGuild);
       let text = this.parseText(message);
       let displayUsername = nickname;
+
+      // Join the display name with double "bold" formatting characters
+      // to prevent unintentional highlights on the IRC side
+      displayUsername = displayUsername.split('').join('\u200d');
```

Comment:
Maybe merge this with line 164? I.e. `let displayUsername = nickname.split('').join('\u200d');`